### ISSUE TITLE
A way to work on OpenWrt 22.05

### ISSUE THREAD

**Issue by david-ng-hk** [benisai/Openwrt-Monitoring#14]
Hi, awesome project, beautiful UI.
I just fond this project, and it work on 22.05
https://github.com/benisai/netify-sqlite/

I think it use something like this
```
ln -s /tmp/dhcp.leases /www/dhcp.html
```

hope this project canwork on OpenWrt v22 too.

**Comment by ginxxxxx** [comment]
the link you shared is owned by the same person, benisai.

Much is not working at all ... 
opkg install prometheus returns unknown package 'prometheus' on openwrt 21

**Comment by benisai** [comment]
> the link you shared is owned by the same person, benisai.
> 
> Much is not working at all ... opkg install prometheus returns unknown package 'prometheus' on openwrt 21

If your opkg location doesnt have prometheus, then you will miss alot of data in grafana. 

![image](https://github.com/user-attachments/assets/89212d66-a7cb-4a3d-a09b-9fdbb7627e51)

**Comment by benisai** [comment]
> ln -s /tmp/dhcp.leases /www/dhcp.html

yes :) so I place the dhcp file in the /www/ folder so it can be accessible via your browser. 192.168.1.1\dhcp.html. It will use this to gather the hostname for IP mapping within the container.

**Comment by benisai** [comment]
> Hi, awesome project, beautiful UI. I just fond this project, and it work on 22.05 https://github.com/benisai/netify-sqlite/
> 
> I think it use something like this
> 
> ```
> ln -s /tmp/dhcp.leases /www/dhcp.html
> ```
> 
> hope this project canwork on OpenWrt v22 too.

try my new project for netify logs

https://github.com/benisai/Openwalla/tree/main
